src/main/java/com/dataloom/authorization/HazelcastAuthorizationService.java

```diff
@@ -55,10 +63,25 @@
     public HazelcastAuthorizationService(
             HazelcastInstance hazelcastInstance,
             AuthorizationQueryService aqs,
-            EventBus eventBus ) {
+            EventBus eventBus,
+            Neuron neuron ) {
         aces = hazelcastInstance.getMap( HazelcastMap.PERMISSIONS.name() );
         this.aqs = checkNotNull( aqs );
         this.eventBus = checkNotNull( eventBus );
+
+        aces.addEntryListener( (EntryUpdatedListener) event -> {
```

Comment:
Good to know. Changes to `HazelcastAuthorizationService` are temporary, just in place to help me visualize how Neuron fits into the bigger picture, how to interact with it, etc. Will add a comment to remove before merging into develop.